### DISCUSSION TITLE
TypeError: Cannot read properties of null (reading 'destroy')

### DISCUSSION THREAD

**Discussion by anhhoangsky** [vercel/community#1288] [Help] [answered]
### Summary

My remix deploy and build done affter solution #1282 1282#discussioncomment-4713736
but have issue
![image](https://user-images.githubusercontent.com/33333758/213136326-ac68fa48-56cc-402b-b36c-b5171d9697b4.png)

**Comment by swarnava** [comment, marked as answer]
Hi, 

We have released a new version of Vercel CLI. Can you remove the environment variable > redeploy and let us know if you are still facing the issue.
